src/Components/Nav/Nav.scss

```
@@ -0,0 +1,78 @@
+.shippingBanner {
+  width: 100%;
+  height: 50px;
+  display: flex;
+  justify-content: center;
+  background-color: #252525;;
+  border: 1px solid
+}
+
+
+.shippingBannerButton {
+  height: 21px;
+  margin: auto;
+  background: transparent;
+  border: 0;
+}
+
+.shippingBannerContent {
+  color: #fffef2;
+}
+
+
+.navHeader {
+  height: 69px;
+  position: relative;
+  display: flex;
+  align-items: center;
+  font-size: 14px;
+}
+
+.navHeaderContainer {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```

```suggestion
.navHeader {
  height: 69px;
  position: relative;
  display: flex;
  align-items: center;
  font-size: 14px;
  
  .navHeaderContainer {
    width: 100%;
    display: flex;
    justify-content: space-between;
    align-items: center;
  }
}
```
이런식으로 sass의 nesting을 이용해서 스타일링을 하시면 좋습니다!

---

네 scss nesting 했습니다